js/views/userPage/UserPage.js

```diff
@@ -0,0 +1,113 @@
+import $ from 'jquery';
+import BaseVw from '../baseVw';
+import loadTemplate from '../../utils/loadTemplate';
+import app from '../../app';
+import Home from './UserPageHome';
+import Store from './UserPageStore';
+import Follow from './UserPageFollow';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    this.tabViewCache = {};
+    this.tabViews = { Home, Store, Follow };
+
+    this.followed = false; // TODO check to see if user is followed by the viewer
+    this.followsYou = true; // TODO check to see if this user follows the viewer
+    this.ownPage = true; // TODO check to see if this is the viewer's page
+  }
+
+  className() {
+    return 'userPage';
+  }
+
+  events() {
+    return {
+      'click .js-tab': 'tabClick',
+      'click .js-followBtn': 'followClick',
+      'click .js-messageBtn': 'messageClick',
+      'click .js-moreBtn': 'moreClick'
+    };
+  }
+
+  tabClick(e) {
+    const targ = $(e.target).closest('.js-tab');
+    this.selectTab(targ);
+  }
+
+  followClick() {
+    // TODO add in follow functionality
+    if (this.followed) {
+      // unfollow this user
+      console.log('unfollow');
+      // do the following as the callback of the unfollow action
+      this.followed = false;
+      this.$followLbl.removeClass('hide');
+      this.$unfollowLbl.addClass('hide');
+    } else {
+      // do the following as the callback of the follow action
+      console.log('follow');
+      this.followed = true;
+      this.$followLbl.addClass('hide');
+      this.$unfollowLbl.removeClass('hide');
+    }
+  }
+
+  messageClick() {
+    // activate the chat message
+    console.log('message');
+  }
+
+  moreClick() {
+    this.$moreableBtns.toggleClass('hide');
+  }
+
+  selectTab(targ) {
+    let tabViewName = targ.data('tab');
+    let tabView = this.tabViewCache[tabViewName];
+    const tabViewType = tabViewName; // the original view name is passed in for the Follow view
```

Comment:
Since the following and follower tabs appear to be identical other than their titles, the plan here is to pass in a flag for the title and use the same view/template for both.